data/mods/gen6/random-teams.js

```diff
@@ -1029,10 +1029,11 @@ class RandomGen6Teams extends RandomTeams {
 	}
 
 	/**
+	 * @param {PlayerOptions} [side]
 	 * @param {number} [depth]
 	 * @return {RandomTeamsTypes["RandomFactorySet"][]}
 	 */
-	randomFactoryTeam(depth = 0) {
+	randomFactoryTeam(side, depth = 0) {
```

Comment:
I don't like having to pass a `side` param everywhere. Can you just stick the seed in the generator object? It should already have one, right?

Reply:
> Can you just stick the seed/PRNG property in the generator object? It should already have one, right?

This change belongs to 25edaa3, which is unrelated to the PRNG, and that's what already happens.

``side`` is only used for the player name. I figured I'd restore the old team generator signature... However, an alternative is a ``setMeta()`` function that would set some ``.meta.playerName``.

Reply:
What's the player name used for?

Reply:
As a seed to create a random team specific to the player

Old Seasonals:
- https://github.com/Zarel/Pokemon-Showdown/blob/old-seasonals/2013/scripts.js#L339
- https://github.com/Zarel/Pokemon-Showdown/blob/old-seasonals/2013/scripts.js#L424
- https://github.com/Zarel/Pokemon-Showdown/blob/old-seasonals/2014/scripts.js#L50

SSB Proposal:
- https://github.com/Zarel/Pokemon-Showdown/blob/5d5b1afde80840f9ea3c2b37488c81fda45f200e/data/mods/ssb/random-teams.js#L650-L653